Source/automap.cpp

```diff
@@ -509,7 +509,20 @@ void DrawAutomapText(const Surface &out)
 		}
 
 		DrawString(out, desc, linePosition);
+		linePosition.y += 15;
+	}
+	switch (sgGameInitInfo.nDifficulty) {
+	case DIFF_NORMAL:
+		strcpy(desc, _("Difficulty: Normal"));
+		break;
+	case DIFF_NIGHTMARE:
+		strcpy(desc, _("Difficulty: Nightmare"));
+		break;
+	case DIFF_HELL:
+		strcpy(desc, _("Difficulty: Hell"));
+		break;
 	}

```

Comment:
Pulling the constant "Difficulty: " text into a format string would allow reusing existing translations for this context, this approach is used in diablo.cpp:1488 (KeyMapper game info action) as well as others.

```suggestion
	string_view difficulty;
	switch (sgGameInitInfo.nDifficulty) {
	case DIFF_NORMAL:
		difficulty = _("Normal");
		break;
	case DIFF_NIGHTMARE:
		difficulty = _("Nightmare");
		break;
	case DIFF_HELL:
		difficulty = _("Hell");
		break;
	}
	CopyUtf8(desc, fmt::format(
	    _(/* TRANSLATORS: {:s} means: Game Difficulty. */ "Difficulty: {:s}"),
	    difficulty
	), sizeof(desc));
```

(untested, might not compile as-is)